todo_app/tests/test_client.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pytest
+import requests
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app.
+    test_app = app.create_app()
+
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+def test_index_page(client):
+
+    response = client.get('/')
```

Comment:
This is an old version of the test which I expect to fail (because there's no patching to handle the calls to Trello) but actually we get away with it because the next function has the same name so it just overrides this and it gets ignored, so let's get rid of this 😄 

```suggestion
```